### PR TITLE
Flutter for iOS — Form input section

### PR DESCRIPTION
**Pull request by rock3r**
This PR is for the eighth section of the work to create the `flutter-for-ios.md` document. Writing by @niamh-power on Novoda's behalf, I am just a proxy.

Following the same rough format as the `flutter-for-android.md` document, with iOS details instead.

The following sections have been created:

* What is the equivalent of a placeholder in a text field?
* How do I show validation errors?

⚠️ **This requires #982 to be merged first!** ⚠️

### CODE REVIEW

File: flutter-for-ios.md [hunk 1]
```diff
@@ -27,3 +27,1720 @@ that are most relevant to your needs.
...
+- `hashCode`
+- `height`
+- `inherit`
+- `letterSpacing`
+- `textBaseline`
+- `wordSpacing`
+
+# Form Input
+
+## What is the equivalent of a placeholder in a text field?
```
Comment (mjohnsullivan) [inline flutter-for-ios.md]: Kind of abrupt way to introduce forms, directly diving in to placeholders. Shouldnt it mention what widgets are using for forms, and reference the TextEditingController

**Comment by rock3r** [comment]
Contents covered in #993, closing this. Will address comments on that one.

File: flutter-for-ios.md [hunk 2]
```diff
@@ -27,3 +27,1720 @@ that are most relevant to your needs.
...
+{% endprettify %}
+
+# Theming and text
+
+## How do I theme an app?
+
+You may have noticed a lot of the examples in this document use a `MaterialApp`
+as their app widget. This provides a base of themed widgets to use in your app.
+However these are more themed to an Android device, so in order to use more iOS
+focused widgets you will need to implement these as needed. You can find the
```
[the remark this answers is not in the document]
Comment (rock3r) [reply flutter-for-ios.md]: Done in #993

File: flutter-for-ios.md (hunk 1 above)
Comment (rock3r) [reply flutter-for-ios.md]: Good point. Added in #993

**Pull request closed without merging**